packages/special-pages/messages/duckplayer/openInfo.notify.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is ` "$schema": "http://json-schema.org/draft-07/schema#”` needed here?

Reply:
Ahh yeah, thanks I'll add it in.